R/class-workbook.R

```diff
@@ -3780,6 +3780,7 @@ wbWorkbook <- R6::R6Class(
       if (length(cols) == 0L) {
         return(self)
       }
+      assert_class(hidden, "logical", arg_nm = "hidden")
```

Comment:
Will this break existing code? Or is 1L == TRUE?

Reply:
Reverted